cmd/get/releases/command.go

```diff
@@ -0,0 +1,85 @@
+package releases
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
+)
+
+const (
+	name  = "releases [release-name]"
```

Comment:
Why is this called a `name` and not a `version`?

Reply:
No real reason, updated to call it version instead.